src/level/robotmain.cpp

```diff
@@ -5650,6 +5652,86 @@ void CRobotMain::ClearInterface()
     m_tooltipName.clear();  // really removes the tooltip
 }
 
+void CRobotMain::NotifyObjectDestroyed(CObject* pObj)
+{
+    switch (pObj->GetType())
```

Comment:
This introduces a bug: delete is spamming errors when a unit is destroyed with WIN or SQUASH; also WIN is used in CodeBattles for the winning team so it shouldn't display any errors at all